roles/internal/Islandora-Devops.alpaca/README.md

```diff
@@ -24,58 +24,181 @@ Where to clone Alpaca to:
 alpaca_clone_directory: /opt/alpaca
 ```
 
-Karaf repo:
+Path to alpaca app jar file:
 ```
-alpaca_karaf_repos:
-  - mvn:ca.islandora.alpaca/islandora-karaf/LATEST/xml/features
+alpaca_jar_path: "/opt/alpaca/islandora-alpaca-app/build/libs/islandora-alpaca-app-2.0.0-all.jar"
 ```
 
-Which features to install:
+Alpaca user:
 ```
-alpaca_features:
-  - islandora-http-client
-  - islandora-connector-broadcast
-  - islandora-indexing-triplestore
-  - islandora-indexing-fcrepo
+alpaca_user: vagrant
 ```
 
-Where Karaf is:
+Alpaca group:
 ```
-alpaca_karaf_dir: /opt/karaf
+alpaca_group: vagrant
```

Comment:
This should be an Ansible default ssh user variable, since it may not be running on Vagrant.